sdk/core/Azure.Core.Expressions.DataFactory/CHANGELOG.md

```diff
@@ -1,27 +1,29 @@
 # Release History
 
-## 1.0.0-beta.7 (Unreleased)
+## 1.0.0 (2024-03-05)
 
 ### Features Added
 
-### Breaking Changes
+- Azure.Core.Expressions.DataFactory is now generally available.
 
-### Bugs Fixed
+### Breaking Changes
```

Comment:
We generally don't record changes between betas as breaking and limit this to just breaking changes between GA versions in order to ensure that we limit noise and GA breaking changes are easily found by scanning.

Can we move this to "Other changes"?

Reply:
Hmm that's news to me.

Reply:
I prefer to not report intra-beta changes as breaking. We want to alert users to changes that they need to be aware when they transition to new versions of GAed libraries. Intra-beta change just add noise IMO.